system/models/users/customer.py

```diff
@@ -1,3 +1,19 @@
-class Customer:
-    def get_discount_category(self):  # return discount category
-        pass  # print the products order
+from system.models.users.user import User
+from system.models.shopping.discount import DiscountCategory
+
+
+class Customer(User):
+    def __init__(self, user_name, password, country_id):
```

Comment:
There should be type casting added on this method. So like this:

def __init__(self, user_name: str, password: str, country_id: int):